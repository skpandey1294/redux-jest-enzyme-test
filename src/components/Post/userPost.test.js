import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from "react-redux";
import configureMockStore from "redux-mock-store";
import UserPost from './userPost';

const mockStore = configureMockStore();
const store = mockStore({});

let wrapped = shallow(
    <Provider store={store}>
        <UserPost />
    </Provider>
)


describe("UserPost Component", () => {

    it("UserPost should render without throwing an error", () => {
        expect(wrapped.debug()).toMatchSnapshot();
    });
});