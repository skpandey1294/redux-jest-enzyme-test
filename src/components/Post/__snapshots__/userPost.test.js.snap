// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserPost Component UserPost should render without throwing an error 1`] = `
"<ContextProvider value={{...}}>
  <Component />
</ContextProvider>"
`;
